examples/metadata-operations/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en">
+  <head>
+    <title>Metadata Operations</title>
+
+    <!-- Live reload script -->
+    <script src="http://localhost:35729/livereload.js?snipver=1"></script>
+
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <script type="module" src="main.js"></script>
+
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/@vertexvis/viewer@0.17.x/dist/viewer/viewer.css"
+    />
+    <script
+      type="module"
+      src="https://unpkg.com/@vertexvis/viewer@0.17.x/dist/viewer/viewer.esm.js"
+    ></script>
+    <script
+      nomodule
+      src="https://unpkg.com/@vertexvis/viewer@0.17.x/dist/viewer.js"
+    ></script>
+  </head>
+  <body>
+    <vertex-viewer id="viewer" class="viewer" config-env="platdev">
```

Comment:
In some of the more recent examples, we've been adding some of our public keys so that developers have a working example out of the gate with 0 configuration. This could be a candidate here to add the super car or vertex-valve.

Reply:
Ok, would be nice to add that to our existing `loadViewerWithQueryParams` so it works consistently across all our examples.